components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -47,8 +42,8 @@ export function StartWorkspaceModal(p: StartWorkspaceModalProps) {
         {/* separator */}
         <div className="border-t border-gray-200 dark:border-gray-800 mt-2 -mx-6 px-6 pt-2">
             <div className="flex">
-                <Tab name='Recent' setSelection={setSelection} selection={selection} />
-                <Tab name='Examples' setSelection={setSelection} selection={selection} />
+                <TabMenuItem name='Recent' selected={selection === 'Recent'} onClick={() => setSelection('Recent')} />
+                <TabMenuItem name='Examples' selected={selection === 'Examples'} onClick={() => setSelection('Examples')} />
```

Comment:
**question**: Could we make this tab menu disappear when creating a new team similar with the specs? Low priority! 🔅

Reply:
Now tracked in https://github.com/gitpod-io/gitpod/issues/4471